docs/source/solvers.rst

```diff
@@ -5,15 +5,6 @@ Solver Routines
 Orbital Localization
 ====================
 
-Molecular orbital localization
-------------------------------
-
-.. autofunction:: quemb.molbe.lo.localize
-
-Crystalline orbital localization
---------------------------------
-
-.. autofunction:: quemb.kbe.lo.localize
```

Comment:
Maybe I'm missing something -- is there a reason why we are removing the autodocs here (and 67-74 below)?

Reply:
these functions do not exist anymore they are now part of the `BE` class and moved there as methods.
(They are documented via `autoclass`)

This was done in #30

Reply:
ah i see! cool!